shell/models/cluster.x-k8s.io.machinedeployment.js

```diff
@@ -100,6 +101,21 @@ export default class CapiMachineDeployment extends SteveModel {
     return machinePools.find(pool => pool.machineConfigRef.name === machineConfigName);
   }
 
+  toggleScaleDownModal( event, showScalePoolPrompt, resources = this ) {
+    const alt = isAlternate(event);
```

Comment:
Please add a comment about we check if the user keep pressed they key `Cmd/Ctrl`

The logic is already confusing as it is, since the function is called `Alt` but the key is not.